src/localization/en.js

```diff
@@ -269,6 +269,41 @@ export default {
       chartsHeader_v1:            `With the new pay, how could your benefits change?`,
       warningMessage_v1:          `This tool is in testing and these numbers might not be right. If they're not, we'd appreciate your feedback.`,
       submitFeedback_v1:          `Submit Feedback`,
+      // added the following snippets Friday 2018_9_7
+      // Benefits Table:
+      benefitsTableTitle_v1:      `Changes`,
+      columnBenefit_v1:           `Benefit`,
+      columnCurrentBenefits_v1:   `Current Benefits`,
+      columnNewEstimate_v1:       `New Estimate`,
+      columnDifference_v1:        `Difference`,
+      rowSNAP_v1:                 `SNAP`,
+      rowSection8_v1:             `Section 8 Housing`,
+      rowTotalBenefits_v1:        `Total Benefits`,
+      rowIncome_v1:               `Income`,
+      rowNetTotal_v1:             `Net Total`,
+      perMonth_v1:                ` / month`,
+      dollarSign_v1:              `$`,
```

Comment:
Am I off-base, or does this use the changes introduced here: https://github.com/codeforboston/cliff-effects/pull/730/files#diff-2961681bf8b9ccd9382824bc4430f707R268?

Reply:
My PR did introduce a subset of these changes (I copied the snippets I needed from the google doc). My vote is on rebasing whichever ends up getting merged second. Feel free to ping me when you get to that point and I'd be happy to do it.